cfg/aws/credentials.go

```diff
@@ -203,6 +205,34 @@ func newStsCredentials(c client.ConfigProvider, roleARN string, region string) *
 	return credentials.NewCredentials(&stsCredentialProvider{regional: regional, partitional: partitional})
 }
 
+var (
+	sourceAccount = os.Getenv(envconfig.AMZ_SOURCE_ACCOUNT) // populates the "x-amz-source-account" header
+	sourceArn     = os.Getenv(envconfig.AMZ_SOURCE_ARN)     // populates the "x-amz-source-arn" header
```

Comment:
So these should not be modifiable after start up?

Reply:
How do these get configured on EC2? The systemd service won't pick up external environment variables.